test/jit.cpp

```diff
@@ -421,9 +421,24 @@ TEST(JIT, ConstEval7) {
     const array f = constant(1, 1);
     const array g = constant(1, 1);
 
-    // I can't think of a good test for this.
     EXPECT_NO_THROW({
         eval(a, b, c, d, e, f, g);
         af::sync();
-      });
+    });
+}
+
+TEST(JIT, IncrementingBinary) {
```

Comment:
Perhaps `VeryLargeTreeEvaluation` instead of `IncrementingBinary` makes more sense as the test name.